packages/commonjs/README.md

```diff
@@ -106,6 +106,13 @@ Default: `true`
 
 If false, skips source map generation for CommonJS modules.
 
+### `transformMixedEsModules`
+
+Type: `Boolean`<br>
+Default: `false`
+
+In case of modules that contain commonjs code you can enable/disable the transformer. Set it to `true` if you know that `require` calls should be transformed, or `false` if you know that the code contains env detection and the `require` should survive the transform.
```

Comment:
```suggestion
In case of ES modules that contain CommonJS elements such as `require` calls, you can enable/disable the transformer. Set it to `true` if you know that `require` calls should be transformed, or `false` if `require` should survive the transform unchanged, which can be useful to ignore environment detection code or when bundling the render process of an Electron application.
```

Reply:
I think it might be confusing to refer to ES modules as just modules here because CommonJS modules are also modules in a way.